src/styles/reset.scss

```
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
```

```suggestion
  font-family: "Roboto", Arial, Helvetica, sans-serif;
```
Add a fallback font if "Roboto" is not found